### DISCUSSION TITLE
incorrect number of stars displayed in tab

### DISCUSSION THREAD

**Discussion by wickedjargon** [community/community#17193] [Other Feature Feedback, Questions, & Ideas]
I have removed all my stars, but my profile still reports that I have 13 stars (see the number after `stars` in the top right tab):

![pic-selected-220523-1914-28](https://user-images.githubusercontent.com/11734208/169919144-fc592a46-9adc-4996-859f-77968c4d9a6a.png)

this inaccurate number is also displayed for other users. You can check my profile here:

https://github.com/wickedjargon?tab=stars

**Comment by wickedjargon** [comment]
In order to fix this problem, I stared a project, and then unstared it, refreshed my profile page, and all the number of stars disappeared. I guess this is no longer an issue for me, but perhaps some testing should be done on weather removing stars always updates this number.

**Reply by user95f85f** [reply]
I think most people don't notice the wrong star count because they've got like 80+ stars. (Assuming they get the wrong star count like we did when we went on a removal spree.)

**Comment by PedanticHacker** [comment]
I am also experiencing this issue. The stars count is incorrect on the `Stars` tab. It is annoying, especially because this is a reoccurring bug.

I wouldn't expect something so simple not being fixed already by a company that is the foundation for worldwide software development.

**Comment by user95f85f** [comment]
>I wouldn't expect something so simple not being fixed already...
we just gotta complain super hard.

**Reply by PedanticHacker** [reply]
Yeah, we gotta nag, there's no other way.

**Comment by AroKol78** [comment]
~I have similarly, 60 ⭐ and it shows the number 59~

---
edited:
on my part, I already have it out of date.
the entire repository has magically disappeared, I had it on my list `stars`.

**Comment by 743859910** [comment]
I have similarly，The webpage displays 476⭐, and the mobile terminal displays 472⭐

**Comment by jfullstackdev** [comment]
data will take time to be updated, particularly when there is cached data, just try logging in in Incognito ( private browsing ) mode and see

**Reply by 743859910** [reply]
That data update time is really long enough.

**Comment by jfullstackdev** [comment]
hopefully this will be seen by one of the Staff members here and take action

**Comment by MrSarno** [comment]
Your profile now correctly shows that you haven't starred any repositories. 🙂

**Comment by jchamlin** [comment]
I am seeing the same thing. I have 20 repositories starred, but only a count of 16 is showing on my Stars tab. From my testing, it appears private repositories that own and that you star (like for the purposes of including them in a stars list) are not counted in the total displayed on the Stars tab. I have 4 private repositories starred (3 I own and 1 that is a fork of another private repository), so that's why I have 20 stars but only showing 16. Starring/unstarring those four private repositories does not cause the Stars tab count to change. It's confusing for sure.

**Comment by cyny39** [comment]
https://github.com/github/choosealicense.com/issues/1195

**Reply by huhtig** [reply]
Leave a reply

**Comment by huhtig** [comment]
~~> In order to fix this problem, I stared a project, and then unstared it, refreshed my profile page, and all the number of stars disappeared. I guess this is no longer an issue for me, but perhaps some testing should be done on weather removing stars always updates this number.
~~
